spec/helpers/rpc_mock/server.lua

```diff
@@ -0,0 +1,199 @@
+local helpers = require("spec.helpers")
+local client = require("spec.helpers.rpc_mock.client")
+local default_cert = require("spec.helpers.rpc_mock.default").default_cert
+
```

Comment:
I strongly recommend you to write a callback and diagram in the comment like

```
diagram:

mock client -> mock server -> tested DP

callback
func1
 -> func2
  -> func3 ...

```

otherwise other developers or reviewers find it hard to understand, today i talked with @chronolaw , he found it hard to understand what the server.lua does here.

Reply:
a good way is that you could write a test case to run this `mock client -> mock server -> tested DP` progress, i find there is no case for `mock server<>tested DP`. Then we could explain the progress with that case, that might be easy for reviewers.